eclair-rpc/src/main/scala/org/bitcoins/eclair/rpc/json/EclairModels.scala

```diff
@@ -317,3 +341,40 @@ implicit val sendResultReads: Reads[SendResult] = Reads[SendResult] { json =>
     }
   }
 }*/
+
+object WebSocketEvents {
+
+  sealed trait Event
```

Comment:
For readability, I'd say it makes more sense to move the trait outside of `WebSocketEvents`, and rename it to match it. 

I.e.

```scala
sealed trait WebSocketEvents
object WebSocketEvents {
  case class PaymentRelayed(
    ...
```